### PR TITLE
ci/windows: disable spool

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
We're not expecting to print anything, and @RPS5' security newsletter
says this is a vector of attack.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: infra/vsts_agent_windows.tf [hunk 1]
```diff
@@ -83,7 +95,7 @@ Set-StrictMode -Version latest
 $ErrorActionPreference = 'Stop'
 
 # Disable Windows Defender to speed up disk access
-Set-MpPreference -DisableRealtimeMonitoring $true
+Set-MpPreference -DisableRealtimeMonitoring $true${local.w[count.index].extra}
```
Comment (cocreature) [inline infra/vsts_agent_windows.tf:98]: what does this change do?

Comment (garyverhaegen-da) [reply infra/vsts_agent_windows.tf:98]: This inserts lines 23-26 for the "no-spool" cohort, while doing no change at all to the default machines, so they don't get killed prematurely.

Arguably I could have put that on the blank line below.

The "mergeable" version of this PR will get back to just the default group, and this change inlined.

Comment (cocreature) [reply infra/vsts_agent_windows.tf:98]: Oh I see, I missed the fact that it has a newline at the beginning so it won’t actually affect this command.

**Review by cocreature — APPROVED** [review]
Thanks!

**Pull request merged**